src/App.js

```diff
@@ -33,7 +43,11 @@ const App = () => {
       <h1>To Do List</h1>
 
       <AddTodoForm onAddTodo={addTodo} />
-      <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
+      {isLoading ? (
+          <p>Loading...</p>
+        ): (
+            <TodoList todoList={todoList} onRemoveItem={removeTodo}/>
```

Comment:
There is an error when trying to remove an item - I believe is a simple naming issue, try to change `onRemoveItem` to `onRemoveTodo`. 

```
<TodoList todoList={todoList} onRemoveTodo={removeTodo} />
      )}
```

Reply:
I was able to add this in! Thank you :)